### PR TITLE
Feature/tech 760: manage custom parameters for test suite

### PR DESCRIPTION
**Pull request by lruzicki**
Ticket:
https://govstack-global.atlassian.net/browse/TECH-760

Changes:
- script for managing custom and default parameters is added to the flow

### CODE REVIEW

File: src/templates/base-generated-config.yaml [hunk 1]
```diff
@@ -19,10 +19,36 @@ jobs:
       - run:
           name: Execute tests
           command: |
+            echo "# Execute application test_entrypoint.sh in background."
```
Comment (lruzicki) [inline src/templates/base-generated-config.yaml:22]: this line will be deleted

File: src/templates/base-generated-config.yaml [hunk 2]
```diff
@@ -19,10 +19,36 @@ jobs:
...
             cd << parameters.example-app-path >> && ./test_entrypoint.sh
+            echo "files inside << parameters.example-app-path >>"
+            cd << parameters.example-app-path >> && pwd
+            cd << parameters.example-app-path >> && ls
+            echo "files inside << parameters.test-suite-path >>"
+            cd << parameters.test-suite-path >> && pwd
+            cd << parameters.test-suite-path >> && ls
+            echo "Check for testCustomParameters.json"
+            # Check for testCustomParameters.json
+            if [ -f << parameters.example-app-path >>/testCustomParameters.json ]; then
```
Comment (lruzicki) [inline src/templates/base-generated-config.yaml:28]: can we keep this statement here? or do we want to set it inside script?

File: src/templates/base-generated-config.yaml [hunk 3]
```diff
@@ -19,13 +19,36 @@ jobs:
...
+              echo "File testCustomParameters.json has been copied to << parameters.test-suite-path >>."
+            else
+              echo "File testCustomParameters.json does not exist in << parameters.example-app-path >>."
+            fi
+
+            echo "# Execute script for replacing placeholders in report files"
+            # Execute script for replacing placeholders in feature files
+            if [ -f << parameters.test-suite-path >>/gherkin_param_manager.sh ]; then
+              chmod u+x << parameters.test-suite-path >>/gherkin_param_manager.sh
+              sudo chmod uo+w << parameters.test-suite-path >>/result
```
Comment (lruzicki) [inline src/templates/base-generated-config.yaml:42]: Are we allowed to use 'sudo' to add permissions to the result directory.

Comment (dborowiecki) [reply src/templates/base-generated-config.yaml:42]: I think it should be fine in this context.

**Review by dborowiecki — APPROVED** [review]

**Pull request merged**